NEWS.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.7] - 2022-10-07
```

Comment:
Please note that we do not necessarily release a new patch version each time that we add features to the library. 

If you do not want to release yet, then you just temporarily use `## Unreleased` in the `NEWS.md` file.

I would not release at this time a new version as I anticipate that we are going to need in the near future some extra changes from the refactorization that we are carrying out. Do you agree?

Reply:
Ok great, I do agree. I'll push the changes.